centreon/www/api/class/centreon_metric.class.php

```
@@ -340,139 +223,333 @@ public function getMetricsDataByMetric()
+     * @return array
+     */
+    public function getMetricsDataByPoller()
+    {
+        global $centreon;
+
+        $userId = $centreon->user->user_id;
+        $isAdmin = $centreon->user->admin;
+
+        // Get ACL if user is not admin
+        if (! $isAdmin) {
+            $acl = new CentreonACL($userId, $isAdmin);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$acl'. (Unused Code Rules)

(UnusedLocalVariable)